content/docs/howto/app-monitor.md

```diff
@@ -0,0 +1,267 @@
+
+Connection credentials will be read from the binding at runtime, with the binding secrets being transformed to system properties with the following the pattern `APPDYNAMICS_<KEY>=<VALUE>`.
+
+The buildpack supports the Java, Node.js & PHP languages.
+
+## Aternity
+
+The [Aternity][aternity] integration is enabled with [bindings][bindings] through the [Aternity Buildpack][bp/aternity]. If a binding of the correct type, `AppInternals`, is provided at build-time then an agent appropriate for the application will be contributed to the application image.
+
+Connection credentials will be read from the binding at runtime, with the binding secrets being transformed to system properties with the following the pattern `<KEY>=<VALUE>`.
+
+The buildpack supports the Java language.
+
+## Azure Application Insights
+
+The [Azure Application Insights][azure application insights] integration is enabled with [bindings][bindings] through the [Azure Application Insights Buildpack][bp/azure-application-insights]. If a binding of the correct type, `ApplicationInsights`, is provided at build-time then an agent appropriate for the application will be contributed to the application image.
+
+Connection credentials will be read from the binding at runtime, with the binding secrets being transformed to system properties with the following the pattern `APPLICATIONINSIGHTS_<KEY>=<VALUE>`.
+
+The buildpack supports the Java and Node.js languages.
+
+## Datadog
+
+The [Datadog][datadog] integration is enabled by setting an environment variable for the [Datadog Buildpack][bp/datadog]. If the environment variable `BP_DATADOG_ENABLED` is set to a truthy value (i.e. true, t, 1 ignoring case) at build-time then an agent appropriate for the application will be contributed to the application image.
+
+Datadog configuration is read from any of the standard Datadog environment variables that are set at runtime. This happens directly, the buildpack does not translate them. You can find the full list of configuration properties in the Datadog documentation [available here](https://docs.datadoghq.com/tracing/setup_overview/setup/java/?tab=containers#configuration).
+
+The Datadog agent requires a side-car agent to be running in addition to the Java agent. This side-car agent runs outside of the buildpack generated image. The [standard Datadog instructions for your container orchestrator of choice](https://docs.datadoghq.com/tracing/setup_overview/setup/java/?tab=containers#configure-the-datadog-agent-for-apm) can be used to install this agent. The Paketo team also has detailed instructions for [various runtimes available here](https://github.com/paketo-buildpacks/datadog/blob/main/docs/).
```

Comment:
```suggestion
The Datadog agent requires a side-car agent to be running in addition to the Java agent. This side-car agent runs outside of the buildpack generated image. The [standard Datadog instructions for your container orchestrator of choice](https://docs.datadoghq.com/tracing/setup_overview/setup/java/?tab=containers#configure-the-datadog-agent-for-apm) can be used to install this agent. The Paketo team also has detailed instructions for [various runtimes available here](https://github.com/paketo-buildpacks/datadog/blob/main/docs/run-on-docker.md).
```
Should this link to here instead? It says various runtimes but there's only Docker instructions there

Reply:
That's future-proofing :)  Hopefuly, we'll have instructions for at least Kubernetes as well.